### DISCUSSION TITLE
How to get 2 tables side by side

### DISCUSSION THREAD

**Discussion by mr-easy** [mljar/mercury#458] [Q&A] [answered]
I have to show two tables (using itables) and I want to keep them side by side because they are small. I tried with basic HTML, and it worked in Jupyter Notebook, but it's not working when in Mercury.
![Screenshot 2024-07-11 112204](https://github.com/mljar/mercury/assets/16632324/e85928d8-8706-4fbb-b063-ff74ab29fe4b)
![Screenshot 2024-07-11 112228](https://github.com/mljar/mercury/assets/16632324/a2a79a98-bf33-4159-b181-9e51636fbca0)

**Comment by pplonski** [comment]
Hi @mr-easy,

thanks for reporting this, your code looks good. Let me check this ...

**Reply by pplonski** [reply, marked as answer]
I don't know the reason, but it looks like when using itables parent `div` is deleted ...

When using plain html in the div, instead itables it all works as expected. Really strange, looks like itables is doing some html manipulation.

I've used `<div style="display:flex"> ... </div>` two times and it is working. Below my screenshots

![image](https://github.com/mljar/mercury/assets/6959032/19fcf640-4b55-4962-85bd-45fb9a87f95f)

![image](https://github.com/mljar/mercury/assets/6959032/f22ded5a-2bb7-43be-96ae-67f77b6d7e85)

**Reply by pplonski** [reply]
@mr-easy please let me know if double `div` solution works for you

**Reply by mr-easy** [reply]
Thanks for the quick update, yeah double div worked!
